frontend/src/Components/AlgInputs/AlgInputsLoadControl.js

```diff
@@ -47,29 +50,95 @@ class AlgInputsLoadControl extends Component {
         };
     }
 
+<<<<<<< HEAD
+    componentDidMount() {
+        countyRes.then(res => {
+            this.setState({
+                counties: res.data,
+            });
+        });
+
+        loadControlPromise.then(res => {
+            if(!res.data.length) {
+                return;
+            }
+            this.setState({
+                result: [{
+                    controlled_load: res.data[0].controlled_load,
+                    uncontrolled_load: res.data[0].uncontrolled_load,
+                }]
+            });
+        });
+
+        // this.useDefaultParameters();
+    }
+
+    /* TODO change default parameters of Load Controll */
+    useDefaultParameters = () => {
+        // TODO: backend * 3
+        // Get default parameter set
+
+        Object.keys(loadControlDefaultParams).forEach(param => {
+            this.setState({
+                [param]: loadControlDefaultParams[param],
+            });
+        });
+
+        // this.setState({
+        //     county: loadControlDefaultParams.county,
+        //     rate_energy_peak: loadControlDefaultParams.rate_energy_peak,
+        //     rate_energy_partpeak: loadControlDefaultParams.rate_energy_partpeak,
+        //     rate_energy_offpeak: loadControlDefaultParams.rate_energy_offpeak,
+        //     rate_demand_peak: loadControlDefaultParams.rate_demand_peak,
+        //     rate_demand_partpeak: loadControlDefaultParams.rate_demand_partpeak,
+        //     rate_demand_overall: loadControlDefaultParams.rate_demand_overall,
+        // });
```

Comment:
Do we need this comment or can we just remove this block of code.

Reply:
removed